roboteam_ai/test/ConditionTests/BallOutOfFieldtest.cpp

```diff
@@ -22,6 +22,7 @@ TEST(BallOutOfFieldTest, it_detects_ball_out_of_field) {
     worldMsg.ball.pos.x = 0.0;
     worldMsg.ball.pos.y = 0.0;
     worldMsg.ball.visible = 1;
+    worldMsg.ball.area = 99999;
```

Comment:
what does the ball area do?

Reply:
Ball area is default 0, now I check if the ball has an area 'not 0', so all balls in world need an area now.

In grsim and ssl vision (or probably in vision/world) the ball always has area 99999, so I just make it that.